.buildkite/pull-requests.json

```diff
@@ -2,11 +2,10 @@
   "jobs": [
     {
       "enabled": true,
-      "pipelineSlug": "logstash-filter-elastic_integration-pull-request-pipeline",
+      "pipelineSlug": "logstash-filter-elastic_integration-pull-request",
       "allow_org_users": true,
       "allowed_repo_permissions": ["admin", "write"],
       "set_commit_status": true,
-      "commit_status_context": "buildkite/logstash-filter-elastic_integration-pull-request-pipeline",
```

Comment:
this will be the message issued on github; IIRC by default it will be `buildkite/<the pipeline name>` but you can give it some other name.